model/modelindexer/indexer.go

```diff
@@ -473,4 +478,8 @@ type Stats struct {
 
 	// BytesTotal represents the total number of bytes that
 	BytesTotal int64
+
+	// AvailableBulkIndexers represents the number of bulk indexers
+	// available for making bulk index requests.
+	AvailableBulkIndexers int64
```

Comment:
"Available bulk indexers" seems like an internal detail of the implementation which I'd rather not expose.

The configuration is "MaxRequests", and there's an existing metric which counts the total number of bulk requests over time. Maybe we should follow that naming? Perhaps "AvailableBulkRequests" (struct field) and "available_bulk_requests" (Elasticsearch field)?